lms/envs/common.py

```diff
@@ -1654,6 +1654,14 @@
     'ROOT_PATH': '/tmp/edx-s3/ora2-responses',
 }
 
+##################### ORA2 Responses Download #######################
```

Comment:
Although, I don't think this settings ever gets used. If you take a look at https://github.com/Stanford-Online/edx-platform/blob/master/lms/djangoapps/instructor_task/models.py#L201